routes/cartRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router  = express.Router();
+
+const {loginRequired} = require("../utils/auth")
+const { cartController } = require('../controllers');
+
+router.post('/',loginRequired, cartController.additionCart);
+router.get('/',loginRequired, cartController.getByCart);
+router.patch('/change', loginRequired,cartController.cartQuantityChange);
+router.delete('/all',loginRequired, cartController.allDeleteCart);
+router.delete('/one',loginRequired, cartController.oneDeleteCart);
```

Comment:
- RESTFull 하게 endpoint를 작성해 주세요.
- 특정 cart 1개를 지우고 싶은 행위를 할려면 아래와 같이 path parmeter로 cart의 id를 받아서 삭제하는 것이 적절해 보입니다.
```suggestion
router.delete('/:cartId',loginRequired, cartController.oneDeleteCart);
```

Reply:
해당 내용은 외부에서 받아서 넣을 수 있도록 변경하였습니다.